language/tools/move-cli/src/package/cli.rs

```diff
@@ -314,6 +321,108 @@ pub fn handle_package_commands(
             config
                 .resolution_graph_for_package(&rerooted_path)?
                 .print_info()?;
+        },
+        PackageCommand::Upload => {
+            let mut upload_request: UploadRequest = Default::default();
+            let mut output = Command::new("git")
```

Comment:
I'm wondering if this can reuse the package manager's code for downloading from github: https://github.com/move-language/move/blob/main/language/tools/move-package/src/resolution/resolution_graph.rs#L517

Reply:
Do you mean we should go with `map_err` instead of `unwrap` like in the code you linked, *OR* do you mean we should try to reuse the function `download_and_update_if_repo`, in which case not necessary because here we are querying info from the local (already fetched) repo.